sdcm/sct_events.py

```diff
@@ -339,6 +349,25 @@ def eval_filter(self, event):
         return result
 
 
+class EventsFilter(BaseFilter):
```

Comment:
write docstring with the modes that the filter supports

Reply:
done